slack/http_client.py

```diff
@@ -33,13 +33,17 @@
 
 def get(method, params):
     url = _build_url(method)
-    response = requests.get(url, params=params, verify=True).json()
-    _raise_error_if_not_ok(response)
-    return response
+    return _parse_response(requests.get(url, params=params, verify=True))
```

Comment:
Minor thing but I think it's easier to read if we can break this down into two lines:
```
response = requests.get(url, params=params, verify=True)
return  _parse_response(response)
```